.eslintignore

```diff
@@ -0,0 +1,9 @@
+node_modules
+dist
+README.md
+COPYING
+src/webrtc
+assets/cache
+assets/client_test
+assets/locales
+assets/public
```

Comment:
I feel like these should be included?

Reply:
For some reason my local file just outright ignored all of `assets` and was not showing as changed lol.

There's no way to set the eslint environment to `browser` for an entire folder, so we'd need to specify it per file which is not good. If we didn't do this, all the preload/inline/plugins would return `no-undef` errors.